docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -0,0 +1,108 @@
+---
+id: plugin-types-usage
+title: Plugin types and usage
+description: Learn about the different types of Grafana plugins, their usage and configuration.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - usage
+  - provisioning
+  - provision
+  - configuration
+  - configure
+sidebar_position: 1
+---
+
+# Grafana plugin types and usage
+
+:::note
+
+This documentation discusses plugin configuration, provisioning, and usage for plugin developers. General information about plugin administration is available at [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+::: 
+## Overview
+
+Grafana plugin development allows for many options depending on the type of user experience you want to create. Whatever your need, there's a SDK-supported plugin type for your intended use:
+- **Data-source plugin** - a connection to a new database or other source of data,
+- **App plugin** - an integrated out-of-the-box experience, or
+- **Panel plugin** - a new way of visualizing data.
+
+:::note
+
+Refer to [Get started](../get-started/get-started.mdx) for instructions on how to quickly scaffold [each type](../get-started/get-started.mdx#what-type-of-plugin-would-you-like) of plugin.
+
+:::
+## Data source plugins
+
+In plugin development, you can create new data source plugins to use Grafana with new databases, similar to existing plugins such as [MongoDB](https://grafana.com/grafana/plugins/grafana-mongodb-datasource/) or [Google BigQuery](https://grafana.com/grafana/plugins/grafana-bigquery-datasource/). Data source plugins may be added on the frontend and [the backend](./backend.md).
+
+### Usage of data-source plugins
+
+Create and configure a Grafana data source when you want to work with data provided by a third-party service for use in Grafana Dashboards, Explore, Alerting, and so on.
+
+Given any installed plugin of type `datasource`, you can create and configure any number of data sources per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations). After you create zero to infinity data sources, they are persisted in Grafana's database. 
+
+:::note
+
+To distinguish a Grafana data source from a data-source plugin, we sometimes refer to the latter as a _data-source instance_, that is, a configured Grafana data source with a `plugin id` type of `datasource`.
+
+:::
+
+### Configuration of data sources
```

Comment:
Not sure, but maybe denote Global configuration of data source plugins to not confuse with section above?

```suggestion
### Global configuration of data source plugins
```

Reply:
if you say so. Changed.

Reply:
done.